### PR TITLE
[MAD-15948] Fixed Motion loader

### PR DESCRIPTION
**Pull request by vmednonogov**
Ticket: MAD-15948

## What does this PR do?

Changes:
- Removed retry stuff workaround from PRs 305 & 312 (reverted back to original code)
- Do not use MotionSetCallback class as default loader because it does not work with platforms (e.g. with iOS)
- Waiting while asset loader  is initialized

## How was this PR tested?

Verified locally on PC, on iOS Jenkins build 27572

### CODE REVIEW

**Comment by Twolewis** [comment]
Before we merge this in I really want QA to hammer on a build.  Can you work that out w/ Mike?

**Review by Twolewis — APPROVED** [review]

**Comment by carbmsanders** [comment]
@Twolewis I had QA attempt to reproduce 30 times, and they could not. Please find a link to the slack conversation within the Jira ticket. This has my seal of approval to be merged.

**Comment by Twolewis** [comment]
Phenomenal - take it away, @vmednonogov !

**Pull request merged**
